rewrite-java-test/src/test/java/org/openrewrite/java/ReorderMethodArgumentsTest.java

```diff
@@ -85,9 +71,8 @@ public void test() {
     @Test
     void reorderArgumentsWithNoSourceAttachment() {
         rewriteRun(
-          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(..)",
-              new String[]{"s", "n"}, new String[]{"n", "s"}, null, null))
-            .cycles(1).expectedCyclesThatMakeChanges(1),
+          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(String,..)",
```

Comment:
had to make the matcher a bit more specific, otherwise it was always matching no matter if we changed the method Type